app/models/webhooks/slack/base.rb

```diff
@@ -45,8 +46,8 @@ def view_discussion_on_loomio(params = {})
     { value: discussion_link(I18n.t(:"webhooks.slack.view_it_on_loomio"), params) }
   end
 
-  def proposal_link(proposal, position = nil)
-    discussion_link position || proposal.name, { proposal: proposal.key, position: position }
+  def proposal_link(event, position = nil)
```

Comment:
Ah, so this wouldn't be an 'event', but a 'model' (A Proposal or Vote is a model, while a NewVote or NewComment is an 'event')